### PR TITLE
New set of Hobbies

### PR DESCRIPTION
**Pull request by ShadeOfDead**
#### Summary
Content "New Hobbies"

#### Purpose of change
Filling out the hobbies with more options included.

#### Describe the solution

This begins fleshing out the Hobbies.

#### Testing
Several starts with the different hobbies, it is functioning correctly.

### CODE REVIEW

**Comment by definitelynotklip** [comment]
Hey, congrats on your first PR! :relaxed:
You need to lint the file you changed so that it is in theme with the styling of this project.

Visit the following link, copy the entire text from the file over, and press the `Lint` button. Copy the linted version then replace it in the file you submitted. This will be done in a new commit. It helps pass the tests so that reviewers may merge your work quicker.
http://dev.narc.ro/cataclysm/format.html

Best of luck!

File: data/json/hobbies.json [hunk 1]
```diff
@@ -91,5 +91,88 @@
...
+    "description": "You whiled your nights away using, building and repairing old Ham Radios.  Now you have no one to talk to.  But you won't give up hope and maybe your electronics knowledge can help you somehow.",
+    "points": 1,
+    "skills": [ { "level": 3, "name": "electronics" } ]
+  },
+  {
+    "type": "profession",
+    "subtype": "hobby",
+    "id": "knitting",
+    "name": "Knitting",
+    "description": "You enjoyed many a day sitting in your rocking chair and knitting mindlessly while watching TV shows.  The TV shows are no more, but you still can enjoy knitting right?",
```
Comment (anothersimulacrum) [inline data/json/hobbies.json:146]:
```suggestion
    "description": "You enjoyed many a day sitting in your rocking chair and knitting mindlessly while watching TV shows.  The TV shows are no more, but you still can enjoy knitting, right?",
```

File: data/json/hobbies.json [hunk 2]
```diff
@@ -91,5 +91,88 @@
...
+    "type": "profession",
+    "subtype": "hobby",
+    "id": "back_yard_grilling",
+    "name": "Back Yard Grilling",
+    "description": "You enjoyed inviting the neighborhood to a backyard get together most weekends.  You, of course, were the grill master.",
+    "points": 1,
+    "skills": [ { "level": 3, "name": "cooking" } ]
   }
-]
+]
```
Comment (anothersimulacrum) [inline data/json/hobbies.json:178]:
```suggestion
]

```

File: data/json/hobbies.json [hunk 3]
```diff
@@ -91,5 +91,88 @@
...
+    "skills": [ { "level": 2, "name": "gun" }, { "level": 2, "name": "shotgun" } ]
+  },
+  {
+    "type": "profession",
+    "subtype": "hobby",
+    "id": "back_yard_grilling",
+    "name": "Back Yard Grilling",
+    "description": "You enjoyed inviting the neighborhood to a backyard get together most weekends.  You, of course, were the grill master.",
+    "points": 1,
+    "skills": [ { "level": 3, "name": "cooking" } ]
```
Comment (LaVeyanFiend) [inline data/json/hobbies.json:176]:
```suggestion
    "skills": [ { "level": 3, "name": "cooking" } ],
    "proficiencies": [ "prof_food_prep", "prof_knife_skills" ]
```

File: data/json/hobbies.json (hunk 2 above)
Comment (ShadeOfDead) [reply data/json/hobbies.json]: done.

File: data/json/hobbies.json (hunk 3 above)
Comment (ShadeOfDead) [reply data/json/hobbies.json]: added.

File: data/json/hobbies.json [hunk 4]
```diff
@@ -91,5 +91,89 @@
...
+    "skills": [ { "level": 4, "name": "swimming" } ]
+  },
+  {
+    "type": "profession",
+    "subtype": "hobby",
+    "id": "arts_and_crafts",
+    "name": "Arts and Crafts",
+    "description": "You loved doing crafts at home.  Hopefully this can aid you in surviving now that the world has gone to hell.",
+    "points": 1,
+    "skills": [ { "level": 2, "name": "fabrication" }, { "level": 2, "name": "tailor" } ]
```
Comment (Maleclypse) [inline data/json/hobbies.json:121]:
```suggestion
    "skills": [ { "level": 1, "name": "fabrication" }, { "level": 2, "name": "tailor" } ]
```

File: data/json/hobbies.json [hunk 5]
```diff
@@ -91,5 +91,89 @@
...
+    "skills": [ { "level": 3, "name": "mechanics" } ]
+  },
+  {
+    "type": "profession",
+    "subtype": "hobby",
+    "id": "ham_radio_operator",
+    "name": "Ham Radio Operator",
+    "description": "You whiled your nights away using, building and repairing old Ham Radios.  Now you have no one to talk to.  But you won't give up hope and maybe your electronics knowledge can help you somehow.",
+    "points": 1,
+    "skills": [ { "level": 3, "name": "electronics" } ]
```
Comment (Maleclypse) [inline data/json/hobbies.json:139]: I'd reduce electronics by one and add one point of speech

File: data/json/hobbies.json [hunk 6]
```diff
@@ -91,5 +91,89 @@
...
+    "skills": [ { "level": 2, "name": "gun" }, { "level": 2, "name": "shotgun" } ]
+  },
+  {
+    "type": "profession",
+    "subtype": "hobby",
+    "id": "back_yard_grilling",
+    "name": "Back Yard Grilling",
+    "description": "You enjoyed inviting the neighborhood to a backyard get together most weekends.  You, of course, were the grill master.",
+    "points": 1,
+    "skills": [ { "level": 3, "name": "cooking" } ],
```
Comment (Maleclypse) [inline data/json/hobbies.json:176]:
```suggestion
    "skills": [ { "level": 2, "name": "cooking" } ],
```

**Review by Maleclypse — COMMENTED** [review]
Thanks for adding this!

File: data/json/hobbies.json [hunk 7]
```diff
@@ -91,5 +91,89 @@
...
+  {
+    "type": "profession",
+    "subtype": "hobby",
+    "id": "knitting",
+    "name": "Knitting",
+    "description": "You enjoyed many a day sitting in your rocking chair and knitting mindlessly while watching TV shows.  The TV shows are no more, but you still can enjoy knitting, right?",
+    "points": 1,
+    "proficiencies": [ "prof_knitting" ],
+    "skills": [ { "level": 2, "name": "tailor" } ]
+  },
```
Comment (LaVeyanFiend) [inline data/json/hobbies.json:150]:
```suggestion
```
We already have this.

**Pull request merged**